doc/pages_samples.md

```diff
@@ -0,0 +1,1011 @@
+  },
+  "next_page": null
+}
+```
+
+Returned from API:
+```json
+{
+  "id": 3,
+  "question_text": "What is your date of birth?",
+  "question_short_name": null,
+  "hint_text": "Hint",
+  "answer_type": "date",
+  "is_optional": null,
+  "answer_settings": {
+    "input_type": "other_date"
+  },
+  "created_at": "2023-01-19T11:03:08.661Z",
+  "updated_at": "2023-01-19T11:17:16.617Z",
+  "form_id": 1,
+  "position": 3,
+  "next_page": null
+}
+```
+
+#### With date of birth set
+
+Sent from admin:
+```json
+{
+  "question_text": "What is your date of birth?",
+  "hint_text": "Hint",
+  "answer_type": "date",
+  "is_optional": true,
+  "answer_settings": {
+    "input_type": "date_of_birth"
+  }
+}
+```
+
+Returned from API:
+```json
+{
+  "id": 3,
+  "question_text": "What is your date of birth?",
+  "question_short_name": null,
+  "hint_text": "Hint",
+  "answer_type": "date",
+  "is_optional": true,
+  "answer_settings": {
+    "input_type": "date_of_birth"
+  },
+  "created_at": "2023-01-19T11:03:08.661Z",
+  "updated_at": "2023-01-19T11:03:08.661Z",
+  "form_id": 1,
+  "position": 3,
+  "next_page": null
+}
+```
+
+### Email address
+There are no options.
+
+Sent from admin:
+```json
+{
+  "question_text": "Email address",
+  "hint_text": "hint",
+  "answer_type": "email",
+  "is_optional": true,
+  "answer_settings": null
+}
+```
+
+Returned from API:
+```json
+{
+  "id": 4,
+  "question_text": "Email address",
+  "question_short_name": null,
+  "hint_text": "hint",
+  "answer_type": "email",
+  "is_optional": true,
+  "answer_settings": null,
+  "created_at": "2023-01-19T11:20:10.557Z",
+  "updated_at": "2023-01-19T11:20:10.557Z",
+  "form_id": 1,
+  "position": 4,
+  "next_page": null
+}
+```
+
+### National Insurance Number
+There are no options.
+
+Sent from admin:
+```json
+{
+  "question_text": "National Insurance Number",
+  "hint_text": "hint",
+  "answer_type": "national_insurance_number",
+  "is_optional": true,
+  "answer_settings": null
+}
+```
+
+Returned from API:
+```json
+{
+  "id": 7,
+  "question_text": "National Insurance Number",
+  "question_short_name": null,
+  "hint_text": "hint",
+  "answer_type": "national_insurance_number",
+  "is_optional": true,
+  "answer_settings": null,
+  "created_at": "2023-01-19T11:27:50.413Z",
+  "updated_at": "2023-01-19T11:27:50.413Z",
+  "form_id": 1,
+  "position": 7,
+  "next_page": null
+}
+```
+
+### Phone number
+There are no options.
+
+Sent from admin:
+```json
+{
+  "question_text": "Phone Number",
+  "hint_text": "hint",
+  "answer_type": "phone_number",
+  "is_optional": true,
+  "answer_settings": null
+}
+```
+
+Returned from API:
+```json
+{
+  "id": 6,
+  "question_text": "Phone Number",
+  "question_short_name": null,
+  "hint_text": "hint",
+  "answer_type": "phone_number",
+  "is_optional": true,
+  "answer_settings": null,
+  "created_at": "2023-01-19T11:21:01.023Z",
+  "updated_at": "2023-01-19T11:21:01.023Z",
+  "form_id": 1,
+  "position": 6,
+  "next_page": 7
```

Comment:
In all the other examples `next_page` is `null`. What's going on here? Is `next_page` null in the other examples because there was no next page? If that's the case, how in this example are we adding a page not to the end of the list?